operating/manage-user-access/configure-ocp-oauth.adoc

```diff
@@ -8,27 +8,19 @@ toc::[]
 [role="_abstract"]
 {ocp} includes a built-in OAuth server that you can use as an authentication provider for {product-title} ({product-title-short}).
 
-[CAUTION]
-====
-The {ocp} OAuth server integration only works when you install {product-title} by using the `roxctl` CLI on {ocp}. It does not work when you install {product-title-short} by using Helm or the {product-title-short} Operator.
-====
+include::modules/configure-ocp-oauth-identity-provider.adoc[leveloffset=+1]
 
 [IMPORTANT]
 ====
-* To disable the {ocp} OAuth server integration, set the `ROX_ENABLE_OPENSHIFT_AUTH` environment variable to `false` in Central:
+* If you use a custom TLS certificate for {ocp} OAuth server, you must add the certificate as a trusted root CA if it is not already trusted. Otherwise, Central cannot connect with the {ocp} OAuth server.
```

Comment:
```suggestion
* If you use a custom TLS certificate for {ocp} OAuth server, you must add the certificate as a trusted root CA. Otherwise, Central cannot connect with the {ocp} OAuth server.
```

Is it possible to drop the "if it is not already trusted" part from this sentence. I understand it would potentially change the meaning of the sentence, however it just seemed a bit long and hard to parse. The other option would be to do potentially do something like, "To use a custom TLS certificate for {ocp} OAuth server, add the certificate as a trusted root CA if it is not already trusted. Otherwise, Central cannot connect with the {ocp} OAuth server."

Reply:
Thanks @sagidlow I've removed the confusing bit and changed it to the following:
```
If you use a custom TLS certificate for {ocp} OAuth server, you must add the CA’s root certificate
to {product-title} as a trusted root CA. 
Otherwise, Central cannot connect to the {ocp} OAuth server.
```